pcsx2-qt/Debugger/Models/BreakpointModel.cpp

```diff
@@ -261,10 +261,9 @@ Qt::ItemFlags BreakpointModel::flags(const QModelIndex& index) const
 	switch (index.column())
 	{
 		case BreakpointColumns::CONDITION:
-			if (is_breakpoint)
-				return Qt::ItemFlag::ItemIsEnabled | Qt::ItemFlag::ItemIsSelectable | Qt::ItemFlag::ItemIsEditable;
-			[[fallthrough]];
+			return Qt::ItemFlag::ItemIsEnabled | Qt::ItemFlag::ItemIsSelectable | Qt::ItemFlag::ItemIsEditable;
 		case BreakpointColumns::TYPE:
+			[[fallthrough]];
```

Comment:
Is the fallthrough needed here since type case doesn't have any specific code?

Reply:
Nope, I can remove that.